File: desing-pattens-1/src/modelo/Orcamento.java
package modelo;

import java.util.ArrayList;
import java.util.List;

public class Orcamento {

	double valor;

	List<Item> itens = new ArrayList<>();

	public Orcamento(double valor) {

		this.valor = valor;
	}

	public double getValor() {

		return valor;
	}

	public List<Item> getItens() {

		return itens;
	}

	public void adicionar(Item item) {

		this.itens.add(item);
	}

	public void comValor(double valor) {

		this.valor = valor;

	}

}

File: desing-pattens-1/src/main/java/modelo/Orcamento.kt
package main.java.modelo

import java.util.ArrayList

class Orcamento(valor: Double) {

    var valor: Double = 0.toDouble()
        internal set

    internal var itens: MutableList<Item> = ArrayList()

    init {

        this.valor = valor
    }

    fun getItens(): List<Item> {

        return itens
    }

    fun adicionar(item: Item) {

        this.itens.add(item)
    }

    fun comValor(valor: Double) {

        this.valor = valor

    }

}
